src/AVFoundation/AVTypes.cs

```diff
@@ -266,4 +266,39 @@ public static RectangleF WithAspectRatio (this RectangleF self, SizeF aspectRati
 		}
 	}
 #endif
+
+	[Mac (10, 10), NoiOS, NoWatch, NoTV]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorSyncInfo {
+		public bool SampleIsFullSync;
+		public bool SampleIsPartialSync;
+		public bool SampleIsDroppable;
+	}
+
+	[Mac (10, 10), NoiOS, NoWatch, NoTV]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorDependencyInfo {
+		public bool SampleIndicatesWhetherItHasDependentSamples;
+		public bool SampleHasDependentSamples;
+		public bool SampleIndicatesWhetherItDependsOnOthers;
+		public bool SampleDependsOnOthers;
+		public bool SampleIndicatesWhetherItHasRedundantCoding;
+		public bool SampleHasRedundantCoding;
+	}
+
+	[Mac (10, 10), NoiOS, NoWatch, NoTV]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorStorageRange {
+		public long Offset;
+		public long Length;
+	}
+
+	[Mac (10, 10), NoiOS, NoWatch, NoTV]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorChunkInfo {
+		public long ChunkSampleCount;
+		public bool ChunkHasUniformSampleSizes;
+		public bool ChunkHasUniformSampleDurations;
+		public bool ChunkHasUniformFormatDescriptions;
```

Comment:
Not sure we need the `Chunk` prefix on every fields. Check if they exists in swift.